### PR TITLE
DCOS-44491 - Changed menuweights in Service docs.

### PR DESCRIPTION
**Pull request by anithamuthu79**
https://jira.mesosphere.com/browse/DCOS-44491

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11, 1.12).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

**Comment by SarahStegall** [comment]
I do not believe we can push to the DSS repo.

**Comment by SarahStegall** [comment]
The service versions you have listed here are not separate documents. For example, the Edge-LB docs are arranged by top version (1.1, 1.2) but patch/interim releases are not documented separated. Thus Edge-LB 1.1.2 would pertain to the changes made in the 1.2 documents, not separate documents. If you have changed the menuweights for Edge-LB 1.1, 1.2, etc then you don't have to worry about subsidiary changes. There is no full 1-to-1 correlation between the version policy page and the list of Mesosphere service documents.

File: pages/services/beta-storage/0.4.0-beta/index.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: DC/OS Storage Service 0.4.0
 title: DC/OS Storage Service 0.4.0
-menuWeight: 0
+menuWeight: 10
```
Comment (imaxxs) [inline pages/services/beta-storage/0.4.0-beta/index.md]: What does menuWeight - 10 means?

File: pages/services/beta-storage/0.5.0-beta/index.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 layout: layout.pug
 navigationTitle: DC/OS Storage Service 0.5.0
 title: DC/OS Storage Service 0.5.0
-menuWeight: 0
+menuWeight: -1
```
Comment (imaxxs) [inline pages/services/beta-storage/0.5.0-beta/index.md]: Why do we have menu weight - 1 here? Does this mean - 0.5.0-beta version will not show up on docs?
If that is correct and 0.4.0-beta menu weight is 10? Should be we disabling/hiding the older version 0.4.0-beta?

File: pages/services/beta-storage/0.4.0-beta/index.md (hunk 1 above)
Comment (anithamuthu79) [reply pages/services/beta-storage/0.4.0-beta/index.md]: @imaxx Here, menu weights are assigned to each file/page to show the hierarchy of occurrence in the left side of the navigation panel in an incremental way. Menu weight start from Zero to 100+
**Example:**
See the left side navigation panel on https://docs.mesosphere.com/services/cassandra/2.4.0-3.0.16/

Page - assigned menu weight

- Release Notes - 10
- Getting Started - 20 
- Operations - 40

.......

File: pages/services/beta-storage/0.5.0-beta/index.md (hunk 2 above)
Comment (anithamuthu79) [reply pages/services/beta-storage/0.5.0-beta/index.md]: @imaxxs Yes, menu weight **-1** is assigned to pages/files that are not supported on https://docs.mesosphere.com/version-policy/#certified-packages-and-dcos-versions
screen shot 2019-02-11 at 12 04 33 pm

As per the above supported versions, I've assigned the following menu weights:
DC/OS Storage Service 0.5.0 - **-1**

**Comment by imaxxs** [comment]
LGTM

**Pull request merged**